samples/sandwiches/Makefile

```diff
@@ -1,9 +1,13 @@
 SWIFT_BIN = /usr/bin
-SWIFT_LIB = /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0
+XCODE_DEV = /Applications/Xcode.app/Contents/Developer
+SWIFT_LIB = $(XCODE_DEV)/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0
 
 SWIFTC = $(SWIFT_BIN)/swiftc
 SDK = `xcrun --sdk macosx --show-sdk-path`
-SWIFTARGS = -v -sdk $(SDK) -L $(SWIFT_LIB) -emit-module -emit-library -enable-library-evolution -emit-module-interface
+SWIFTARGS = -emit-library -enable-library-evolution -emit-module -emit-module-interface \
+	-sdk $(XCODE_DEV)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk \
```

Comment:
you might have to 
* move `11.1` to a variable right after the `XCODE_*` ones; or
* use `MacOSX.sdk` which should be a symlink to `MacOSX11.1.sdk` for your Xcode (and 11.3.sdk for mine)

I prefer the latter but I recall Apple broke it in the past... still likely to require less updates over time

Reply:
Good idea!